core-tests/src/test/java/org/slf4j/impl/StaticLoggerBinder.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.slf4j.impl;
+
+import org.mule.runtime.core.internal.logger.CustomLoggerFactory;
+
+import org.slf4j.ILoggerFactory;
+import org.slf4j.LoggerFactory;
+import org.slf4j.spi.LoggerFactoryBinder;
+
+
+public class StaticLoggerBinder implements LoggerFactoryBinder {
```

Comment:
Add a jdoc, indicating this class and indicating that this will only be used in core-tests.